flow/e2e/bigquery/peer_flow_bq_test.go

```diff
@@ -706,14 +706,18 @@ func (s PeerFlowE2ETestSuiteBQ) Test_Types_BQ() {
 
 	srcTableName := s.attachSchemaSuffix("test_types_bq")
 	dstTableName := "test_types_bq"
-
+	createMoodEnum := "CREATE TYPE mood AS ENUM ('happy', 'sad', 'angry');"
+	_, enumErr := s.pool.Exec(context.Background(), createMoodEnum)
+	if enumErr != nil && !strings.Contains(enumErr.Error(), "already exists") {
```

Comment:
https://pkg.go.dev/github.com/jackc/pgerrcode can check if the code is DuplicateObject